demos/README.md

```diff
@@ -0,0 +1,35 @@
+![buildah logo](https://cdn.rawgit.com/projectatomic/buildah/master/logos/buildah-logo_large.png)
+
+# Buildah Demos
+
+The purpose of these demonstrations is twofold:
+
+1. To help automate some of the tutorial material so that Buildah newcomers can walk through some of the concepts.
+2. For Buildah enthusiasts and practitioners to use for demos at educational presentations - college classes, Meetups etc.
+
+It is assumed that you have installed Buildah and Podman on you machine.
+
+    $ sudo dnf -y install podman buildah
+
+## Building from scratch demo 
+
+filename: `buildah-scratch-demo.sh`
+
+This demo builds a container image from scratch. The container is going to inject a bash shell script and therefore requires the installation of coreutils and bash.
+
+Please make sure you have installed Buildah and Podman. Also this demo uses Quay.io to push the image to that registry when it is completed. If you are not logged in then it will fail at that step and finish. That's fine. If you wish to login to Quay.io before running the demo, then it should push to your repository successfully.
```

Comment:
suggestion.  I'd drop the "That's fine." sentence.   Also suggest "should push" to "will push".

Reply:
+1